src/main/java/com/ksm/wstbackoffice/controller/AddressController.java

```diff
@@ -0,0 +1,44 @@
+package com.ksm.wstbackoffice.controller;
+
+import com.ksm.wstbackoffice.entity.Address;
+import com.ksm.wstbackoffice.service.AddressService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "address")
```

Comment:
Please read about rest api best practice in order to provide correct uri

Reply:
done